LiteCore/Support/SecureRandomize.hh

```diff
@@ -38,6 +38,51 @@
     }
 
     #define SECURE_RANDOMIZE_AVAILABLE 1
+    
+#elif defined(_CRYPTO_MBEDTLS)
+
+    #include <mbedtls/entropy.h>
+    #include <mbedtls/ctr_drbg.h>
+
+#ifdef MBEDTLS_NO_PLATFORM_ENTROPY
+    #include <Windows.h>
+    #include <bcrypt.h>
+    static int uwp_entropy_poll(void *data, unsigned char *output, size_t len,
+        size_t *olen)
+    {
+        NTSTATUS status = BCryptGenRandom(NULL, output, len, BCRYPT_USE_SYSTEM_PREFERRED_RNG);
+        if (status < 0) {
+            return(MBEDTLS_ERR_ENTROPY_SOURCE_FAILED);
+        }
+
+        return 0;
+    }
+#endif
+
+    static int initialized = 0;
+    static mbedtls_entropy_context entropy;
+    static mbedtls_ctr_drbg_context ctr_drbg;
+    static inline void SecureRandomize(fleece::slice s) {
+        if (!initialized) {
```

Comment:
This idiom for one-time initialization isn't thread-safe. Use `std::call_once` instead.